app/code/Ludmila/LSNewClasses/view/frontend/templates/homework11_block.phtml

```diff
@@ -0,0 +1,28 @@
+<div class="homework11">
+    <h2>Files:</h2>
+    <?php
+    foreach ($block->giveFileList() as $filename) {
```

Comment:
Please use foreach (...) : .... endforeach; style in template. Please refer the examples in Magento core. Example:
```
<?php foreach ($block->giveFileList() as $filename): ?>
  <p><?php echo __('%s was modified on %s:', $filename, date('F d Y H:i:s.', filemtime($filename)) ); ?><br/>
<?php endforeach;?>
```
P.S. Just a codestyle example, please check if it works as expected,